enip_backend/export/structs.py

```diff
@@ -0,0 +1,145 @@
+from enum import Enum
+from pydantic import BaseModel, Field
+from datetime import datetime
+from humps import camelize
+
+from typing import Dict, Optional, List, Union
+
+
+def to_camel(string):
+    return camelize(string)
+
+
+# From: https://medium.com/analytics-vidhya/camel-case-models-with-fast-api-and-pydantic-5a8acb6c0eee
+class CamelModel(BaseModel):
```

Comment:
Oh wow this is really cool! kind of mind-expanding to read through this file and technique!